test/spec/books/emptyText.json

```diff
@@ -12,4 +12,4 @@
     "title": "The Lord of the Rings: The Fellowship of the Ring.",
     "text": "An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring."
```

Comment:
Title of the file says emptyText.json, but I don't think the file is empty. Maybe I'm missing something.